src/structures/MessageEmbed.js

```diff
@@ -180,6 +186,7 @@ class MessageEmbed {
       : null;
 
     /**
+     * Represents the footer field of a MessageEmbed
```

Comment:
```suggestion
     * Represents the footer of a MessageEmbed
```
Footer isnt really a field, it has no title, and the value doesnt support much formatting, it should be named independent of fields

Reply:
I think `author field` is a bit misleading too, but i cant think of a better name considering `the author of a MessageEmbed` is misleading too.

Reply:
Yeah its a tricky part.